account/user.go

```diff
@@ -165,3 +165,18 @@ var TestUser = User{
 	Email:    "testdeveloper@testalm.io",
 	Identity: TestIdentity,
 }
+
+// TestIdentity2 only creates in memory obj for testing purposes
+var TestIdentity2 = Identity{
```

Comment:
Thanks for spotting this, @hectorj2f ! You're absolutely right about putting test-related variables in the right `account_test` package. I blame myself for copying-pasting too quickly the `TestIdentity` variable into ` `TestIdentity2` one :-/